kratos/python/add_kratos_parameters_to_python.cpp

```diff
@@ -78,7 +95,7 @@ void  AddKratosParametersToPython()
     .def("ValidateAndAssignDefaults",&Parameters::ValidateAndAssignDefaults)
```

Comment:
minor: just before merging, you could pass the clang-format (or astyle) on these files.

Reply:
If I merge the new format files, the pull-request needs to be approved again.